core/server/models/member.js

```diff
@@ -154,6 +154,11 @@ const Member = ghostBookshelf.Model.extend({
         return options;
     },
 
+    searchQuery: function searchQuery(queryBuilder, query) {
+        queryBuilder.where('name', 'like', `%${query}%`);
+        queryBuilder.orWhere('email', 'like', `%${query}%`);
```

Comment:
This all looks great! Only question is whether `like` is enough, or whether we need to use `ilike` (case insensitive) ?